panther_analysis_tool/data_model.py

```diff
@@ -0,0 +1,125 @@
+"""
+Panther Analysis Tool is a command line interface for writing,
+testing, and packaging policies/rules.
+Copyright (C) 2020 Panther Labs Inc
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as
+published by the Free Software Foundation, either version 3 of the
+License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+import os
+import logging
+import tempfile
+from typing import Any, Callable, Dict, List
+
+from jsonpath_ng import Fields
+from jsonpath_ng.ext import parse
+
+from .util import id_to_path, import_file_as_module, store_modules
+
+_DATAMODEL_FOLDER = os.path.join(tempfile.gettempdir(), 'datamodels')
+
+E_NO_DATA_MODEL_FOUND = "a data model hasn't been specified for log type"
+
+# constants used to extract data from data model
+NAME = 'name'
+PATH = 'path'
+METHOD = 'method'
+
+
+# Temporary alias for compatibility
+get_logger = logging.getLogger
```

Comment:
Logging configuration is handled in [`main`](https://github.com/panther-labs/panther_analysis_tool/blob/e11228f7d2c1115a3cfde6fec01c7cb6a4a767be/panther_analysis_tool/main.py#L1396). Note that `getLogger` is a function and is called when the `DataModel` is created, so it will acquire the centralized logging configuration correctly.

Or is it something else that you were concerned about maybe?